next_rails.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.homepage     = "https://github.com/fastruby/next_rails"
   spec.license       = "MIT"
 
-  spec.required_ruby_version = ">= 2.0"
+  spec.required_ruby_version = ">= 2.0", "< 3.5"
```

Comment:
I thought it would be ideal to block supported Ruby versions to make sure users can rely on it, happy to remove it if you don't find it valuable tho